src/installer/pkg/archives/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <EnsureRuntimeIdentifierSet>true</EnsureRuntimeIdentifierSet>
```

Comment:
Why is this needed?

Reply:
The Archives package uses the RID to determine the default archive name.